mungegithub/mungers/sync/issue-sync.go

```diff
@@ -32,14 +32,9 @@ const (
 	// JenkinsBotName is the name of kubekins bot
 	JenkinsBotName = "k8s-bot"
 	priorityPrefix = "priority/P"
-	// PriorityP0 represents Priority P0
-	PriorityP0 = Priority(0)
-	// PriorityP1 represents Priority P1
-	PriorityP1 = Priority(1)
-	// PriorityP2 represents Priority P2
-	PriorityP2 = Priority(2)
-	// PriorityP3 represents Priority P3
-	PriorityP3 = Priority(3)
+
+	// PriorityFailingTest represents a failing or flaking test
+	PriorityFailingTest = Priority(2)
 )
```

Comment:
I do not think the number is actually used for anything. It was setup to integrate closely with the auto-prioritization code. We could change the type to string and just treat the priority as a constant. That would require some updates to the interface.